src/components/Tile.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const Tile = ({
+  children,
+  type,
+  padding
+}) => {
+  const flexClasses = classnames({
+    'uk-tile': true,
+    [`uk-tile-${type}`]: type,
+    [`uk-tile-${padding}`]: padding
+  })
+
+  return (
+    <div className={flexClasses}>
+      {children}
+    </div>
+  )
+}
+
+Tile.propTypes = {
+  children: PropTypes.oneOfType([
```

Comment:
Do you have some documentation about his ? I usually set the children to be a `func` IIRC (If I Rembember Correctly)

Reply:
`PropTypes.element` in the doc !